meteor/client/ui/Shelf/DashboardPanel.tsx

```diff
@@ -527,16 +605,17 @@ export function getUnfinishedPieceInstancesReactive (currentPartInstanceId: Part
 	}
 
 	// Convert to array of ids as that is all that is needed
-	const unfinishedPieceInstanceIds: { [adlibId: string]: PieceInstanceId[] } = {}
-	_.each(_.groupBy(prospectivePieces, (piece) => piece.piece.adLibSourceId), (grp, id) => unfinishedPieceInstanceIds[id] = _.map(grp, instance => instance._id))
+	const unfinishedPieceInstanceIds: { [adlibId: string]: PieceInstance[] } = {}
+	_.each(_.groupBy(prospectivePieces, (piece) => piece.piece.adLibSourceId), (grp, id) => unfinishedPieceInstanceIds[id] = _.map(grp, instance => instance))
```

Comment:
@thomasslee97 whats the reason for this change? I cant find anywhere that doing anything with the contents, only some checks on the length of the array values